cpp.md

```diff
@@ -27,6 +28,7 @@ int main() {
 * `cout`: to print the output.
 * `main()`: main function is the entry point of any C++ program.
 * `return 0`: To end the main function
+* `endl`: to change line of the output
```

Comment:
This is not widely used, please remove

Reply:
@onecompiler-ops  I have removed "endl" from the 2 places where I had added it. Please review it and merge my pull request. Thank you!